src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,50 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+
+  // storing input in str
+  let str = text;
+
+  // checking if not string then throws error else continues
+  if (typeof str !== "string") {
+    throw "Please enter a valid string";
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.

Reply:
Ok bond noted